### ISSUE TITLE
Deploy Reference Documentation (ALDoc) action missing symbols and xml documentation not showing

### ISSUE THREAD

**Issue by louagej** [microsoft/AL-Go#883]
I added ALDoc to my `settings.json` file (with `be`as supported country):

![image](https://github.com/microsoft/AL-Go/assets/7229592/fc0fdd56-71aa-464d-8942-80479f31fbf3)

The CI/CD action deployed the generated documentation:

![image](https://github.com/microsoft/AL-Go/assets/7229592/0301c1e8-73dd-45ac-a8c1-c8332295e227)

But the static site does not have all information I expected:

1. __MissingTypeSymbol__

> The symbols for the parameter of a function isn't found

This is my procedure in AL:
```AL
    procedure SetStatementNoWithYearPrefix(var CODAStatement: Record "CODA Statement")
```

And this is the output on the generated Docs:

![image](https://github.com/microsoft/AL-Go/assets/7229592/adaf199b-7db6-4086-90ab-89d8a48f29c7)

2. AL XML Documentation does not show

This is the doccumentation in AL:
```AL
    /// <summary>
    /// Set Statement No. With Year Prefix.
    /// Banks reset Statement No. every year. To prevent duplicate Statement No. in BC, we need to add year prefix to Statement No.
    /// </summary>
    /// <param name="CODAStatement">VAR Record "CODA Statement".</param>
```

And this is the output on the generated Docs:

![image](https://github.com/microsoft/AL-Go/assets/7229592/fe2b2484-3ee0-449b-aba6-4cd26e809366)

**Comment by freddydk** [comment]
And the "CODA Statement" record - is that in the same app? or in a dependent app?
Is in in a app from the same repository or a different repo?

**Comment by louagej** [comment]
Hi Freddy,

this is in the `be` localisation

**Comment by freddydk** [comment]
got it - sounds like a bug - will investigate

**Comment by freddydk** [comment]
I can definitely repro this, this is a bug. We will have to populate the a cache folder with referenced apps and hand that over to aldoc.

**Comment by freddydk** [comment]
This reproes for symbols in any base apop
https://microsoft.github.io/bcsamples-bingmaps.pte/reference/bingmaps.pte/Codeunit/BingMaps-Geocode.html

**Comment by RothDominic** [comment]
If we add the parameter -c (packagecache) where the value is the path to the alpackages, the MissingTypeSymbols-Issue is gone.

Current call in AL-Go:
![image](https://github.com/user-attachments/assets/59d5634b-ac38-4af8-b9b7-43d8a95b2243)

How we get it working in powershell:
![image](https://github.com/user-attachments/assets/7d142cca-e9fb-48c8-9124-eec6b45a53f9)

**Comment by louagej** [comment]
I don't think we can currently control this -c parameter with setup in our project.

**Comment by navdotnetreqs** [comment]
Not sure if related; "ALDoc - Referenced module not loaded. Name:'Application'"
https://github.com/microsoft/AL/issues/7965

**Comment by navdotnetreqs** [comment]
@freddydk any updates on this? I see bingmaps still has  __MissingTypeSymbol__.

**Comment by freddydk** [comment]
Haven't had the time to prioritize this yet unfortunately.
Will try to prioritize bug fixes and documentation more over the upcoming period
